docs/xls-30d-amm/transaction-types/ammdelete.md

```diff
@@ -0,0 +1,56 @@
+---
+html: ammcreate.html
+parent: transaction-types.html
+blurb: Create a new Automated Market Maker for trading a given pair of assets.
+labels:
+  - AMM
+status: not_enabled
+---
+# AMMDelete
+[[Source]](https://github.com/gregtatcam/rippled/blob/amm-fixes/src/ripple/app/tx/impl/AMMDelee.cpp "Source")
+
+<embed src="/snippets/_amm-disclaimer.md" />
+
+Delete an empty [Automated Market Maker](../automated-market-makers.md) (AMM) instance that could not be fully deleted automatically.
+
+**Tip:** The [AMMWithdraw transaction](./ammwithdraw.md) automatically tries to delete an AMM, along with associated ledger entries such as empty trust lines, if it withdrew all the assets from the AMM's pool. However, if there are too many trust lines to the AMM account to remove in one transaction, it may stop before fully removing the AMM. Similarly, an AMMDelete transaction removes up to a maximum number of trust lines; in extreme cases, it may take several AMMDelete transactions to fully delete the trust lines and the associated AMM. In all cases, the AMM ledger entry and AMM account are deleted by the last such transaction.
```

Comment:
```suggestion
Normally, an [`AMMWithdraw` transaction](./ammwithdraw.md) automatically tries to delete an AMM and all associated ledger entries when it withdraws all the assets from the AMM's pool. However, if there are too many trust lines to the AMM account to remove in one transaction, it may stop before fully removing the AMM. Similarly, an `AMMDelete` transaction removes up to a maximum number of trust lines; in extreme cases, it may take several `AMMDelete` transactions to fully delete the trust lines and the associated AMM. In all cases, the AMM ledger entry and AMM account are deleted by the last such transaction.
```
Would it be better to specify the 512 limit?

Reply:
Agree, may as well put the 512 limit here. Reworded slightly more than suggested.